nixos/tests/installed-tests/default.nix

```diff
@@ -0,0 +1,68 @@
+# NixOS tests for gnome-desktop-testing-runner using software
+
+{ system ? builtins.currentSystem,
+  config ? {},
+  pkgs ? import ../../.. { inherit system config; }
+}:
+
+with import ../../lib/testing-python.nix { inherit system pkgs; };
+with pkgs.lib;
+
+let
+
+  callInstalledTest = pkgs.newScope { inherit makeInstalledTest; };
+
+  makeInstalledTest =
+    { tested               # Package to test. Needs to have an installedTests output
```

Comment:
Alignment is not diff-friendly.

Reply:
I did something different with the comments so they don't need to be aligned like this.